js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx

```diff
@@ -157,9 +169,54 @@ export const RunsFeedTable = ({
   }
 
   return (
-    <Box flex={{direction: 'column', gap: 8}} padding={{vertical: 12}} style={{height: '100%'}}>
+    <Box
+      flex={{direction: 'column', gap: 8}}
+      padding={{vertical: 12}}
+      style={scroll ? {height: '100%'} : {}}
+    >
       {actionBar}
       {content()}
     </Box>
   );
 };
+
+export const RunsFeedTableWithFilters = ({
+  filter,
+  actionBarComponents,
+}: {
+  filter: RunsFilter;
+  actionBarComponents?: React.ReactNode;
+}) => {
+  const {entries, paginationProps, queryResult} = useRunsFeedEntries(filter, 'all', true);
+  const refreshState = useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
+
+  function content() {
+    if (queryResult.error) {
+      return <RunsFeedError error={queryResult.error} />;
+    }
+    if (queryResult.loading && !queryResult.data) {
+      return (
+        <Box flex={{direction: 'column', gap: 32}} padding={{vertical: 8}} border="top">
+          <LoadingSpinner purpose="page" />
```

Comment:
Consider `SpinnerWithText`. I continue to wage war against giant `page` Spinners...

Reply:
Ahh will swap this out, good call

Reply:
Going to roll this into the PR stacked on this one, since there are other tweaks coming 🙏